How to Conduct a Successful Interview.md

```diff
@@ -0,0 +1,39 @@
+# How to Conduct a Successful Interview
+
+When writing a blog post or article, sometimes it’s useful to include an expert opinion or personal quote to boost your credibility and to improve engagement with your content. If you’ve been assigned to conduct an interview or if you think it could benefit your writing task, follow these guidelines to successfully conduct a professional interview.
+
+- **Ask, don’t expect.**
+
+Generally, it’s best to never assume someone is free. In an email or a Slack message, ask the person politely if they would be interested in answering some questions. When sending the first message, outline what you’re working on and the topic you would like to interview them about so they know what they’re agreeing to from the beginning.
+
+- **Come prepared.**
+
+Once you’ve arranged a time to meet with your interviewee, be sure to prepare for it. Outline all the questions you need answered before the interview, and even memorize some of them so you can work them into the conversation naturally.
+
+- **Ask the right questions.**
+
+Make sure you know exactly who you’re talking to and what their area and level of knowledge is — avoid asking them something they don’t know the answer to. Ask open-ended questions rather than yes/no questions, so they have the opportunity to elaborate on their ideas. Try not to ask only what, but also why, so later on you can explain their ideas well in your written work.
+
+Here’s a few example questions that apply to different types of interviews.
+
+1. What experience do you have with []?
+2. Can you explain how [] works? How do you use it in your company/job/research?
+3. What works about [], and what doesn’t?
+4. Why do you use [] over other products/ideas/platforms on the market?
+5. Would you recommend []? Which groups would it be helpful to?
+
+Obviously, every interview is different. Re-work these questions to apply to your interview topic, or come up with your own if these wouldn’t make sense in your case.
+
+- **Record the interview and take notes.**
+
+You will need to document the answers to your questions. It’s possible to write everything down in real-time, but it’s easy to miss things, and it can be rude to constantly ask someone to repeat themselves. Ask them at the beginning of the interview if they are comfortable being recorded, so you can go back after the interview and transcribe it. During the interview, take notes on the main ideas for you to reference later.
+
+- **Let them speak.**
```

Comment:
The importance of pausing - when you don't talk for a long time, they're likely to fill in the time with more information or experiences they might not have. that kind of stuff.